src/vs/editor/contrib/codeAction/browser/media/action.css

```diff
@@ -30,8 +30,8 @@
 	z-index: 5; /* make sure we are on top of the tree items */
 	content: "";
 	pointer-events: none; /* enable click through */
-	outline: 0px solid; /* we still need to handle the empty tree or no focus item case */
-	outline-width: 0px;
+	outline: 0px solid !important; /* we still need to handle the empty tree or no focus item case */
+	outline-width: 0px !important;
```

Comment:
I think we'll still need some of the importants here. For some reason, in OSS when clicking on a disabled action, there won't be a border. However, in stable and insiders, there will be the blue selection border (ie, if you `Ctrl + r` when nothing in highlighted).